routes/api/jiraRouter.js

```diff
@@ -0,0 +1,33 @@
+const express = require("express");
+const JiraApi = require("jira-client");
+const atob = require("atob");
+
+const router = express.Router();
+
+/* POST call to create new issue. */
+router.post("/issue", async (req, res) => {
+  /* eslint-disable no-console */
+  console.log("Inside create new issue");
+  /* eslint-enable no-console */
+  try {
+    const jira = new JiraApi({
+      protocol: "https",
+      host: req.body.jiraHost,
+      username: atob(req.body.username),
+      password: atob(req.body.password),
```

Comment:
We talked offline just a bit about this.  We should be receiving the username/password as a base64 encoded string for basic authentication (on the authentication header)  instead of as part of the request body.

Reply:
Made changes to consume authentication header in all routes. Also changed jira router to use REST API instead of npm package since it doesn't support authentication header.